pandas/core/dtypes/astype.py

```diff
@@ -258,6 +258,11 @@ def astype_is_view(dtype: DtypeObj, new_dtype: DtypeObj) -> bool:
     -------
     True if new data is a view or not guaranteed to be a copy, False otherwise
     """
+    if dtype.kind in "iufb" and dtype.kind == new_dtype.kind:
```

Comment:
`int64` to `Int64` (numpy to masked) is okay to short cut here?

Reply:
yeah this is definitely a view